src/backend/tcop/postgres.c

```diff
@@ -5930,3 +5930,8 @@ log_disconnections(int code, Datum arg __attribute__((unused)))
 					port->user_name, port->database_name, port->remote_host,
 				  port->remote_port[0] ? " port=" : "", port->remote_port)));
 }
+
+inline bool
```

Comment:
Other functions in this file has comment on usage. 
Of course, the function is self-explanatory but it would be nice to add comment to conform with coding standards.

Reply:
Thx, done.